src/js/entities-service/entities/clinicians.js

```diff
@@ -24,7 +24,11 @@ const _Model = BaseModel.extend({
     return Radio.request('entities', 'roles:model', this.get('_role'));
   },
   can(prop) {
-    return this.get('access') === 'manager';
+    /* istanbul ignore next */
+    return (_DEVELOP_ && !sessionStorage.getItem('cypress'))
```

Comment:
this'll make it so that local development always has access to everything all the time, but prod builds won't be affected.  _so_ we can use the e2e library and/or cypress to test different user accesses, or temporarily comment out this line during local development.  Figure this is better than constantly overriding this for testing.